### PR TITLE
Arrow Hitbox

### PR DESCRIPTION
**Pull request by Neha968**
Arrow needs to be drawn with a different location rectangle than the rectangle that is used by the collision handler.

### CODE REVIEW

File: sprint0/sprint0/Weapons/Arrow.cs
Comment (SnobbyDragon) [inline sprint0/sprint0/Weapons/Arrow.cs:61]: this looks like the same rectangle as Location

File: sprint0/sprint0/Weapons/Arrow.cs
Comment (SnobbyDragon) [inline sprint0/sprint0/Weapons/Arrow.cs:37]: is this supposed to have slightly different values? would it would be helpful to just crop arrow if transparent pixels are causing the bigger hitbox?

File: sprint0/sprint0/Weapons/Arrow.cs
Comment (SnobbyDragon) [inline sprint0/sprint0/Weapons/Arrow.cs:74]: this does not seem different from what is originally assigned

File: sprint0/sprint0/Weapons/Arrow.cs
Comment (SnobbyDragon) [inline sprint0/sprint0/Weapons/Arrow.cs:78]: this doesn't seem different either

**Review by SnobbyDragon — CHANGES_REQUESTED** [review]

File: sprint0/sprint0/Weapons/Arrow.cs
Comment (Neha968) [reply sprint0/sprint0/Weapons/Arrow.cs:78]: width and height get flipped when the arrow is facing east or west. The entire issue that the arrow when facing east or west needs to get drawn with the rectangle of (width, height) but the collision box needs to detect with rectangle of (height, width)

File: sprint0/sprint0/Weapons/Arrow.cs
Comment (Neha968) [reply sprint0/sprint0/Weapons/Arrow.cs:37]: it's not transparent pixels, its when the arrow gets rotated to shoot east or west, the hitbox the collision detect uses doesn't get rotated with it

File: sprint0/sprint0/Weapons/Arrow.cs
Comment (Neha968) [reply sprint0/sprint0/Weapons/Arrow.cs:61]: it is for when it is facing north or south, but not east or west

File: sprint0/sprint0/Weapons/Arrow.cs
Comment (SnobbyDragon) [reply sprint0/sprint0/Weapons/Arrow.cs:78]: ok makes sense. can we move this to the constructor since I think we only need to set width/height once depending on the direction? I can just approve for now if you want to do that later

**Review by SnobbyDragon — APPROVED** [review]
see comment in thread

**Pull request merged**
